glean_parser/util.py

```diff
@@ -253,6 +253,10 @@ def fetch_remote_url(url: str, cache: bool = True):
     Fetches the contents from an HTTP url or local file path, and optionally
     caches it to disk.
     """
+    # Include the Python version in the cache key, since caches aren't
+    # sharable across Python versions.
+    key = (url, str(sys.version_info))
```

Comment:
This isn't strictly related, but I did discover it while testing on a bunch of different Python versions locally (since they share the same cache files).